RIOT-OS/RIOT
# Kconfig: Expose USB configurations

### Contribution description
This PR moves configuration macros of USB to the `CONFIG_` name space and exposes them to Kconfig.

I also added a Kconfig to `examples/usb_minimal` to have VID/PID defaults only for that application. ~~Currently the PR depends on #12913 (97065a8), because~~ I added a check in the application's Makefile to not set the VID/PID via CFLAGS when Kconfig is already being used for that.

### Testing procedure
- By default `examples/usb_minimal` should use all the default configurations as always.
- Running `make menuconfig` should allow to configure USB. When ran in `examples/usb_minimal` VID/PID should have default values. If the default values are changed the warning message should not appear.

### Issues/PRs references
~~Depends on #12913~~
Part of #12888
